msal-auth-node/README.md

```diff
@@ -0,0 +1,120 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- Node.js
+page_type: sample
+name: "Node.js application that makes a request to a protected API"
+description: "This Node.js API protects its own protected endpoint using JWT scope validation."
+products:
+- azure
+- azure-active-directory
+urlFragment: ms-identity-docs-code-api-nodejs
+---
+-->
+
+# Node.js | web API | access control (protected routes) | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This Node.js application uses the Express web framework. The app has a single route that requires an access token. The access token will be automatically validated by MSAL:
+- A missing or invalid (expired, wrong audience, etc) token will result in a `401` response.
+- An otherwise valid token without the proper scope will result in a `403` response.
+- A valid token with the proper scope of `Greeting.Read` will be accepted, and the API will return a "Hello, world" message.
+
+```console
+$ curl http://localhost:8080/ -H "Authorization: Bearer {valid-access-token}"
+Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
+
+```
+## Prerequisites
```

Comment:
~~~suggestion
Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
```

## Prerequisites
~~~